src/modelmanager.cpp

```diff
@@ -895,7 +899,7 @@ Status ModelManager::getPipeline(std::unique_ptr<ovms::Pipeline>& pipelinePtr,
     return status;
 }
 
-const CustomNodeLibraryManager& ModelManager::getCustomNodeLibraryManager() const {
```

Comment:
WHy this const removal is needed?

Reply:
For tests, we mock loading libraries without a need to specify config.json

Reply:
Todo: use library manager initialized outside of model manageer

Reply:
Done